# The FAIR RNA Manifesto

Too many tools to choose from? Help us promote our guidelines and accompanying
practical guide for encouraging the community to collaboratively develop fewer,
better and FAIRer software in RNA bioinformatics!

## Background

Advances in experimental methodologies used in RNA research have led to an
explosion in data volumes and corresponding analysis tools and workflows alike,
with dozens of software choices being available for tasks such as single cell
sequencing analysis or the estimation of transcript expression from RNA-Seq
data. Apart from the sheer amount of available software, the way in which
academic research is conducted, with severe time constraints and a quick
turnover of personnel that rarely consists of trained developers, severely
exacerbates the situation: code bases are not maintained, badly documented,
untested and/or not made publicly available. Consequently, end users find it
difficult to decide which software to use and new software has a hard time
penetrating a crowded market. We propose to challenge this situation with the
FAIR-RNA manifesto, a set of guidelines that aims to concentrate resources on
the development of fewer, higher-quality tools with broad community support.

## Resources

* [Manifesto](MANIFESTO.md)
* [Practical guide for developers](PRACTICAL_GUIDE.md)
* Survey for software users and developers (coming soon...)

## Versioning

The FAIR-RNA manifesto adopts a versioning scheme that is inspired by
[Semantic Versioning](https://semver.org/). Major, minor and patch versions
have the following meaning:

* Major version change: Manifesto guidelines have been modified.
* Minor version change: Other parts of the manifesto (e.g., the preamble or the
  guideline annotations) have been changed.
* Patch version change: Any other resource has been modified.

> Note that the versioning scheme will only come into effect once version 1.0.0
> is released. Up until that point, the manifesto and other resources are to be
> considered as beta releases.

## Contributing

FAIR-RNA is a community effort. The more people behind it, the more people we
will reach. Learn [how to get involved](CONTRIBUTING.md).

## Contributors

In alphabetical order:

* [Angela Chan](https://github.com/achan33)
* [Matthew Crum](https://github.com/mcrum8)
* [Christina Herrmann](https://github.com/ninsch3000)
* [Alex Kanitz](https://github.com/uniqueg)
* [Brenda Miao](https://github.com/bmiao10)
* [Kimberly Olney](https://github.com/olneykimberly)
* [Mathieu Quesnel-Vallières](https://github.com/mathieuq)
* [Joël Simoneau](https://github.com/simojoe)

## Acknowledgements

We would like to thank the following organizations and people:

* [hackseq](https://www.hackseq.com/rna) for giving us a chance to kick-start
  the project during the [2020 hackseqRNA event](https://www.hackseq.com/rna).
* the [iRNA-COSI](https://irnacosi.org/) network for hosting and supporting the
  project.
* Melissa Wilson, Arizona State University, for agreeing to help us gaining IRB
  approval for using our survey results.
* Mihaela Zavolan, University of Basel, and Yoseph Barash, University of
  Pennsylvania, for their continued support of the project
